.osparc/jupyter-neuron/metadata.yml

```diff
@@ -1,28 +1,22 @@
-name: JupyterLab Math (Python+Octave)
+name: JupyterLab NEURON
 thumbnail: https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Jupyter_logo.svg/1024px-Jupyter_logo.svg.png
 description:
   "[JupyterLab](https://jupyter.org/) with **math** kernels in [octave](https://www.gnu.org/software/octave/index)
```

Comment:
is the octave part correct?

Reply:
Yes. We still have the Octave kernel (as in the previous version of jupyter-neuron).